### DISCUSSION TITLE
Installation failed Security Onion 2.3.52

### DISCUSSION THREAD

**Discussion by avinashKumarYadav** [Security-Onion-Solutions/securityonion#4459] [Unsupported Versions]
Hi All,

while installing the Security Onion 2.3.52 getting below mentioned errors:

While checking the /root/sosetup.log
![image](https://user-images.githubusercontent.com/80050262/121296562-826f7600-c90e-11eb-9c67-3775fd14f522.png)

then checked the /root/error.log
![image](https://user-images.githubusercontent.com/80050262/121296856-f7db4680-c90e-11eb-8b9a-509d2877753b.png)

Below is the configuration i am using for SO:
![image](https://user-images.githubusercontent.com/80050262/121297005-41c42c80-c90f-11eb-8846-89e13379712c.png)

Help me to install it.

**Comment by rmcclint** [comment]
I've been working on an upgrade for several days, I will offer some help from my limited experience!  
You need to look more closely at sosetup.log.  First, The fatal error might not actually be what is reported in error.log. Second, The line in error.log exists in sosetup.log -- find that line and read back to see what command was run.  It may help diagnose the issue.   I've seen instances where so-setup-network reported  failure because apt returned an error code, but the error.log noticed something completely different.  Almost every issue I've had has been with apt, because I'm attempting an upgrade from old SO, but I see you are on redhat so I don't have much else to offer.

**Comment by Git-Me-Some-Hub** [comment]
You will have to look a bit deeper in the error logs, but my sosetup.log had this listed: 

`https://repo.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#60 - "Peer's Certificate issuer is not recognized."
Trying other mirror.
It was impossible to connect to the CentOS servers.
This could mean a connectivity issue in your environment, such as the requirement to configure a proxy,`

It ended up that I had my Webfilter blocking the install reaching out to mirrors to download packages.
